src/build/get_build_parameters.mjs

```diff
@@ -29,8 +28,15 @@ export function getBuildParameters(buildParameters) {
   let {
     _: [platform],
     buildMode,
+    networkStack,
+    stagingPercentage,
+    sentryDSN,
```

Comment:
This is the casing we usually prefer at Google: https://google.github.io/styleguide/cppguide.html#General_Naming_Rules 
```suggestion
    sentryDsn,
```

Reply:
But I argue we should remove this flag from this PR.

Reply:
Ah right. Will change.

I will note that personally I don't like this. I think it's unclear. This way `DSN` is obviously an acronym, instead of an abbreviation for, say, `Dishing`/`Dimension` or who knows what